static/src/javascripts-legacy/projects/common/modules/email/run-checks.js

```diff
@@ -30,6 +30,7 @@ define([
     Promise
 ) {
     var emailInserted = false;
+    var compliantOutbrain = false;
```

Comment:
Seems a bit weird storing this property here - could we make this a method/property in the outbrain module and call it when we need it?